orc8r/cloud/Makefile

```diff
@@ -125,15 +126,14 @@ lint_tools:
 ######################
 
 swagger: tools swagger_directories $(SWAGGER_LIST)
-	cp $(MAGMA_ROOT)/orc8r/cloud/go/models/swagger-common.yml $(SWAGGER_V1_ROOT)/$(SWAGGER_COMMON)
-	combine_swagger --in=$(SWAGGER_V1_TMP_GEN) --common=$(SWAGGER_V1_ROOT)/$(SWAGGER_COMMON) --out=$(SWAGGER_V1_YML)
-	rm -rf $(SWAGGER_V1_TMP_GEN)
-	rm $(SWAGGER_V1_ROOT)/$(SWAGGER_COMMON)
+	cp $(MAGMA_ROOT)/orc8r/cloud/go/models/swagger-common.yml $(SPECS_DIR)/$(COMMON_DIR)/$(SWAGGER_COMMON)
+	combine_swagger --in=$(SPECS_DIR) --common=$(SPECS_DIR)/$(COMMON_DIR)/$(SWAGGER_COMMON) --out=$(SWAGGER_V1_YML)
 	swagger validate $(SWAGGER_V1_YML)
 
 swagger_directories:
-	rm -rf $(SWAGGER_V1_TMP_GEN)
-	mkdir $(SWAGGER_V1_TMP_GEN)
+	rm -rf $(SPECS_DIR)
+	mkdir $(SPECS_DIR)
+	mkdir $(SPECS_DIR)/$(COMMON_DIR)
```

Comment:
Do both mkdir commands in a single line using the `-p` flag

Reply:
Oo I didn't know I could do that.